pages/installation.md

```
@@ -313,3 +314,29 @@ So, first install the extension and then update your `settings.json` to use Expe
+cat > start_lexical.sh << 'EOF'
+#!/bin/bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+exec "$DIR/expert_linux_amd64" --stdio
+EOF
+chmod +x start_lexical.sh
+```
+
+Replace `expert_linux_amd64` with the appropriate binary name for your system.
+
+Then, go to `Settings -> Extensions -> Lexical` and enter the folder path in the `Server: Release path override` section:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)